src/ApiManagement.sln

```diff
@@ -0,0 +1,100 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
yep, that is the plan, and will be in the next PR. For now, this PR was just to get the functionalities done and less chance to conflict with other people's PR